mesop/bin/bin.py

```diff
@@ -30,13 +30,45 @@
 def main(argv: Sequence[str]):
   if len(argv) < 2:
     print(
-      """\u001b[31mERROR: missing command-line argument to Mesop application.\u001b[0m
+      """\u001b[31mERROR: missing command-line argument to Mesop.\u001b[0m
 
-Example command:
+Example run command:
 $\u001b[35m mesop file.py\u001b[0m"""
     )
     sys.exit(1)
 
+  if argv[1] == "init":
+    if len(argv) > 3:
+      print(
+        f"""\u001b[31mERROR: Too many command-line arguments for mesop init.\u001b[0m
+
+Actual:
+$\u001b[35m mesop {" ".join(argv[1:])}\u001b[0m
+
+Re-run with:
+$\u001b[35m mesop init file.py\u001b[0m"""
```

Comment:
[question] Do you think people would interpret file.py literally?

-----

I think this goes back to one of the Github issues about the CLI help not being to helpful

Reply:
I'm not sure. I thought about including the argument they provided, but it might not even be a python filename either. I think we can go with this for now and if people get confused we can update this.

Even if they literally do file.py, it's not that bad, we will tell them to run `mesop file.py` and then I think they'll get the idea.